cluster/gce/container-linux/configure-helper.sh

```diff
@@ -1201,6 +1201,10 @@ function start-kube-addons {
   fi
   if [[ "${NETWORK_POLICY_PROVIDER:-}" == "calico" ]]; then
     setup-addon-manifests "addons" "calico-policy-controller"
+
+    # Replace the cluster cidr.
+    local -r calico_file="${dst_dir}/calico-policy-controller/calico-node.yaml"
+    sed -i -e "s@__CLUSTER_CIDR__@${CLUSTER_IP_RANGE:-'10.244.0.0/16'}@g" "${calico_file}"
```

Comment:
I don't understand the default value here. It doesn't match the value in config-default.sh (https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-default.sh#L89) and I'm wondering if this script should just fail if the CIDR isn't set rather than falling back to a value that will likely result in a non-functional cluster.